package.json

```diff
@@ -4,7 +4,7 @@
   "description": "An app that actually takes note...lol",
   "main": "index.js",
   "scripts": {
-    "start": "npm run build:start",
+    "start": "node ./server/build/index.js",
```

Comment:
Any particular reason why you changed the script?

Reply:
No response yet?

Reply:
I realized that Heroku runs the build script eventually.

Reply:
Ohh owkk.